packages/google_maps_flutter/example/lib/map_ui.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async' show Future;
```

Comment:
can you create a button that sets this new map style, and can toggle between this and the default map style?

Reply:
Done